tests/test_search_deed.py

```diff
@@ -9,14 +9,14 @@ def setUp(self):
     @with_context
     @with_client
     def test_search_deed_post(self, client):
-        res = client.post('/searchdeed/search', data={'deed_reference': '38'})
+        res = client.post('/searchdeed/search', data={'borrower_token': '38'})
```

Comment:
borrower token is generated as a hex but only stored as a string so for test purposes it can be anything as the search is mocked